pkgs/tools/compression/bzip2/default.nix

```diff
@@ -1,38 +1,28 @@
 { stdenv, fetchurl
```

Comment:
you need to add back linkstatic, another derviation requires enabling it.

Reply:
Yeah, that's the Darwin issue. I figured I would wait until I knew more about it.